scipy/stats/mstats_basic.py

```diff
@@ -605,13 +637,20 @@ def kendalltau(x, y, use_ties=True, use_missing=False, method='auto'):
         time may grow and the result may lose some precision.
         'auto' is the default and selects the appropriate
         method based on a trade-off between speed and accuracy.
+    alternative : {'two-sided', 'less', 'greater'}, optional
+        Defines the alternative hypothesis. Default is 'two-sided'.
+        The following options are available:
+
+        * 'two-sided': the rank correlation is nonzero
+        * 'less': the rank correlation is negative (less than zero)
+        * 'greater':  the rank correlation is positive (greater than zero)
 
     Returns
     -------
     correlation : float
         Kendall tau
     pvalue : float
-        Approximate 2-side p-value.
+        p-value
```

Comment:
Need to fix this (write a better description) here and probably in stats version. Also need to test mstats alternative interface if we're going to make `alternative` a public parameter in mstats.